client/src/components/brushableHistogram/clamp.js

```diff
@@ -0,0 +1,3 @@
+export default function clamp(val, rng) {
+  return Math.max(Math.min(val, rng[1]), rng[0]);
+}
```

Comment:
given how generic this is, the file should be moved into the `src/util/` directory